css.md

```diff
@@ -0,0 +1,171 @@
+# CSS & Less Guide
+
+This style guide aims to provide the ground rules for CFPB's CSS, such that it's highly readable and consistent across different developers on a team. 
+
+## Table of Contents
+
+1. [General Practices](#general-practices)
+1. [Property Order](#property-order)
+1. [Naming Conventions](#naming-conventions)
+1. [Property values](#property-values)
+1. [Media Queries](#media-queries)
+1. [Less](#less)
+    - [Less General Practices](#less-general-practices)
+    - [Selectors and Nesting](#selectors-and-nesting)
+    - [Variables](#variables)
+
+## General Practices
+
+- Use soft-tabs with a two space indent
+- One line per selector
+- Use a single space after the colon
+- Always end property declarations with a semicolon
+- Put spaces after `:` in property declarations
+- Put spaces before `{` in rule declarations
+
+## Property Order
+
+Box model properties go first, in order from inside the box out, followed by positioning.
+
+Remaining properties are in alphabetical order.
+
+```css
+.selector {
+    /* Display & Box Model */
+    display: inline-block;
+    box-sizing: border-box;
+    width: 100px;
+    height: 100px;
+    padding: 10px;
+    border: 10px solid #333;
+    margin: 10px;
+    overflow: hidden;
+
+    /* Positioning */
```

Comment:
Let's add `float: left;` and `clear: left;` to the top of the positioning group, so it's not ambiguous whether those go in the first or second grouping.